domains/connor.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "A website to show off my projects and or just blogs, etc etc.",
+  "repo": "https://github.com/ZeroTwoDevs/connor.github.io",
+  "owner": {
+    "username": "Connor",
+    "email": "halil_ismail@yahoo.com"
+  },
+  "record": {
+    "CNAME": "zerotwodevs.github.io/connor.github.io"
```

Comment:
CNAME records have to be a hostname and you can only have `<your-username>.github.io`. You can create a different user account or an organization and create a github pages repo from there

Reply:
Oooh! I'm sorry this stuff is so confusing, is https://connordevs.github.io/ right?